README.md

```diff
@@ -134,30 +151,167 @@ or in the case of an error:
 ```ts
 type UpdateProfileResponse = {
   error: string;
-  message: string;
 };
 ```
 
-This route lets someone perform partial updates to their profile. If `name` or
+This route lets the user perform partial updates to their profile. If `name` or
 `nft` is `null`, that field is cleared. If either is `undefined` or omitted,
-nothing happens to that field.
+nothing happens to that field. The name must be unique, at least 1 character
+long, at most 32 characters long, and only contain alphanumeric characters,
+periods, and underscores. The NFT must be owned by any of the profile's public
+keys. If the NFT has no image, it will fail.
+
+When creating a profile for the first time, a set of chain IDs can optionally be
+provided. If provided, the public key will be registered for all of these
+chains. If no chain IDs are provided, the public key will only be registered for
+the chain used to sign the request.
```

Comment:
Would this route be called when a user creates a profile for the first time?

Reply:
yep, it's all just one route

Reply:
Worth saying that explicitly (apologies if you did and I missed it).